lib/MusicBrainz/Server/Data/EditNote.pm

```diff
@@ -106,6 +113,27 @@ sub add_note
     }
 }
 
+sub find_by_recipient {
+    my ($self, $recipient_id, $limit, $offset) = @_;
+
+    my $query = <<'EOSQL';
+        SELECT editor, edit, text, post_time
+          FROM edit_note
+         WHERE editor != $1
+           AND edit IN (SELECT id FROM edit WHERE editor = $1)
+         ORDER BY post_time DESC, edit DESC
```

Comment:
This will be difficult performance-wise for editors with many edits.

Though I guess reosarevok, drsaunde and company will continue to rely on email anyway. ;-)